daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -64,6 +64,17 @@ object Speedy {
     def popEnv(count: Int): Unit =
       env.subList(env.size - count, env.size).clear
 
+    def stackTrace(): ArrayList[Location] = {
```

Comment:
A comment on this method explaining that it gets the callstack from the continuation stack would be nice.